### PR TITLE
HPCC-17288 DOCS:New Scope Options for R plugin

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17288 DOCS:New Scope Options for R plugin
Update documentation for Embedded R code.
NOTE: Initial commit, expecting comments. 

@Michael-Gardner please review
@JamesDeFabia please review

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [ ] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [ ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:

Not fully tested yet, expecting feedback. 
Testing will occur after review.

### CODE REVIEW

File: docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 1]
```diff
@@ -3220,111 +3220,40 @@ split_words('Once upon a time');
...
+            <para>Install the plugin.</para>
 
-            <para>To Install:</para>
+            <para><emphasis role="bold">RPM-based systems: </emphasis>Install
+            using <emphasis>yum install</emphasis>.</para>
 
-            <programlisting>sudo R CMD INSTALL <emphasis role="bold">Rcpp_0.11.0.</emphasis>tar.gz
-</programlisting>
+            <para><emphasis role="bold">Debian-based systems:
+            </emphasis>Install using <emphasis>apt-get install
```
Comment (Michael-Gardner) [inline docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: For debian you might want to say "Install using <emphasis>dpkg -i</emphasis> followed by <emphasis>apt-get install -f</emphasis>.

File: docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 2]
```diff
@@ -3439,7 +3368,7 @@ SUM(NOFOLD(s1b + s2b), a);
             <para>Additional examples of HPCC code can be found at:</para>
 
             <para><ulink
-            url="https://github.com/hpcc-systems/HPCC-Platform/tree/master/testing/ecl/embedjava.ecl">https://github.com/hpcc-systems/HPCC-Platform/tree/master/testing/ecl/embedjava.ecl</ulink></para>
+            url="https://github.com/hpcc-systems/HPCC-Platform/tree/master/testing/ecl/embedjava.ecl">https://github.com/hpcc-systems/HPCC-Platform/tree/master/testing/ecl/embedjava.ecl*</ulink></para>
```
Comment (Michael-Gardner) [inline docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: testing/regress/ecl/embedR.ecl

**Review by Michael-Gardner — COMMENTED** [review]
Couple small changes

**Comment by g-pan** [comment]
Revised and added another line derived from the parent HPCC-17280 @Michael-Gardner please review and in the case of the new text, please review for accuracy/relevance.

**Comment by richardkchapman** [comment]
I can't say I'm a fan of the change from plugin to plug-in. to me, plug-in feels like an adjective (as in "plug-in hybrid car") but plugin feels more like a noun which is what we want.

This change looks ok as far as it goes but needs much more detail on the scope options.

**Comment by g-pan** [comment]
@richardkchapman please review. Scope content added.

File: docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml [hunk 3]
```diff
@@ -225,7 +225,7 @@
         </listitem>
 
         <listitem>
-          <para>Select the security Plugin for each service that requires a
+          <para>Select the security Plug-in for each service that requires a
```
Comment (Michael-Gardner) [inline docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml]: Plug-in -> Plugin

**Review by Michael-Gardner — COMMENTED** [review]
Minor change that Richard pointed out. Otherwise looks good.

**Comment by richardkchapman** [comment]
@gpan back to you. Scope content looks good

File: docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml (hunk 3 above)
Comment (g-pan) [inline docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml]: This instance is outside of the scope of this issue. Note I opened https://track.hpccsystems.com/browse/HPCC-17400 specifically for these other occurrences.

File: docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml [hunk 4]
```diff
@@ -3215,118 +3215,83 @@ split_words('Once upon a time');
...
+          <para><emphasis role="bold">PERSIST</emphasis> - This option
+          controls how long such a shared global scope will persist and
+          exactly how far it will be shared.</para>
 
-            <para>You will need both Rcpp and RInside libraries in order for
-            the R-embedding to work properly. The library installation
-            packages are named with the version number appended so you should
-            search for Rcpp_ and RInside_ to obtain the latest version for
-            your system.</para>
+          <para>The GLOBALSCOPE The value passed to GLOBALSCOPE can be any
```
Comment (JamesDeFabia) [inline docs/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml]: Extra words in this sentence?
The GLOBALSCOPE The value passed to GLOBALSCOPE can be any...
s/b 
The value passed to GLOBALSCOPE can be any

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
the rest looks good

**Review by JamesDeFabia — APPROVED** [review]
looks good now

File: docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml (hunk 3 above)
Comment (richardkchapman) [inline docs/Installing_and_RunningTheHPCCPlatform/Inst-Mods/hpcc_ldap.xml]: But you have introduced the incorrect version with this pull request - that makes it inside the scope!

**Comment by richardkchapman** [comment]
@g-pan Ok apart from one minor comment - but please squash before I merge.

**Comment by g-pan** [comment]
squashed and ready

**Pull request merged**